src/swap/central/changehero.ts

```diff
@@ -255,13 +255,13 @@ export function makeChangeHeroPlugin(
 
     const quoteAmount =
       request.quoteFor === 'from'
-        ? await request.fromWallet.nativeToDenomination(
+        ? await request.fromWallet.convertNativeToDenominated(
```

Comment:
As mentioned in the core PR, we want to move these utilities into this repo, and then it would be `nativeToDenomination(request.fromWallet, request.nativeAmount, request.fromTokenId)`